tests/jballerina-debugger-integration-test/src/test/java/org/ballerinalang/debugger/test/remote/BallerinaTestRemoteDebugTest.java

```diff
@@ -27,19 +27,21 @@
 import java.io.File;
 
 import static org.ballerinalang.debugger.test.utils.DebugUtils.findFreePort;
+import static org.ballerinalang.debugger.test.utils.TestUtils.balServer;
+import static org.ballerinalang.debugger.test.utils.TestUtils.testProjectBaseDir;
 
 /**
  * Test class to test positive scenarios of remote debugging ballerina test command.
  */
-public class BallerinaTestRemoteDebugTest extends BaseTestCase {
+public class BallerinaTestRemoteDebugTest extends DebugAdapterBaseTestCase {
```

Comment:
Why have we extended remote debugging tests from `DebuggerAdapterBaseTestCase`? remote debugging tests are not using the debug adapter.

Reply:
We have moved all the `BaseTestCase` methods to `DebugAdapterBaseTestCase` and completely removed `BaseTestCase` class. Therefore earlier whatever classes extended with `BaseTestCase` are now extended with `DebugAdapterBaseTestCase`

Reply:
Then we need to rename the base class name, as like I mentioned, remote debugging tests are NOT based on debugger adapter

Reply:
Changed the name back to `BaseTestCase`